metadata-ingestion/tests/unit/test_protobuf_util.py

```diff
@@ -0,0 +1,302 @@
+import pytest
+
+from datahub.ingestion.extractor.protobuf_util import (
+    ProtobufSchema,
+    protobuf_schema_to_mce_fields,
+)
+from datahub.metadata.schema_classes import ArrayTypeClass
+
+SCHEMA_WITH_SINGLE_MESSAGE_EMPTY_MESSAGE = """
+syntax = "proto3";
+
+message Test1 {
+}
+"""
+
+SCHEMA_WITH_SINGLE_MESSAGE_SINGLE_FIELD = """
+syntax = "proto3";
+
+message Test2 {
+    string field_1 = 1;
```

Comment:
Hello, should string field_1 = 1 appear in the fields in line 142 in this file? Why I can't find it in the generated fields in line 142.

Reply:
I'm not sure I understand this comment; it appears on line 145.